lte/gateway/c/core/oai/lib/secu/secu_defs.h

```diff
@@ -36,6 +36,19 @@ int derive_key_nas(
     algorithm_type_dist_t nas_alg_type, uint8_t nas_enc_alg_id,
     const uint8_t* kasme_32, uint8_t* knas);
 
+int derive_5gkey_ausf(
+    uint8_t* ck_ik, uint8_t* snni, uint8_t* ak_sqn, uint8_t* kausf);
+
+int derive_5gkey_seaf(uint8_t* kausf, uint8_t* snni, uint8_t* kseaf);
```

Comment:
no longer needed as they are handled on the subscriberdb side.

Reply:
comment addressed.
removed derive_5gkey_ausf  & derive_5gkey_seaf